random_fields/geostatistical_cpt_interpretation.py

```diff
@@ -0,0 +1,682 @@
+            elif 'Matern' in str(self.gpr.kernel):
+                self.vertical_scale_fluctuation = gpr_length_scales[self.v_dim] * np.sqrt(1 / 2)
+            mask = np.arange(len(gpr_length_scales)) != self.v_dim
+            self.anisotropy = gpr_length_scales[mask] / gpr_length_scales[self.v_dim]
+
+
+class ElasticityFieldsFromCpt:
+    """Class for the automatic generation of fields of elastic parameters (vs, rho, G0 and E)
+
+        Attributes:
+            - poisson_ratio (float): poisson ratio.
+            - cpt_file_folder (str): path of the folder containing the CPT files (.gef)
+            - x_ref (float): physical x-coordinate of the origin of the model
+            - y_ref (float): physical y-coordinate of the origin of the model
+            - orientation_x_axis (float):  orientation of the x-axis of the model (azimuth, clockwise g=from the North)
+            - based_on_midpoint (str): If True, ignores x_ref and y_ref and places origin of the model in
+                the center of the set of CPT locations.
+            - max_conditioning_points (int): Maximum number of data points to use in the calibration of the
+                geostatistical model.
+            - porosity (float): porosity of the soil
+            - water_density (float): density of the water
+            - return_property (str): property to return through attribute `ElasticityFieldsFromCpt.generated_field`.
+                Options are `young_modulus`, `rho`, `vs`, `G0`
+            - seed (int): Seed for the `:class:RandomFields`
+            - coordinates_sampled_conditioning (npt.NDArray[np.numpy64]): coordinates of the thinned data used for the
+                calibration of the geostatistical model.
+            - conditioning_data (class:`CPT_data`): instance of the `class:CPT_data` class.
+            - conditional_sample_index (npt.NDArray[int]): Indices of the sample from the conditioning data, selected
+                for the calibration.
+            - trans_model_rho (class:`MarginalTransformation`): Transformation model of the marginal distribution
+                of rho.
+            - trans_model_vs (class:`MarginalTransformation`): Transformation model of the marginal distribution of
+                shear wave velocity vs.
+            - geostat_model (class:`GeostatisticalModel`): Geostatistical model
+            - young_modulus (npt.NDArray[np.float64]): generated conditioned random field of elastic modulus
+                 `young_modulus = 2 * G0 * (1. + poisson_ratio)`
+            - solid_density (npt.NDArray[np.float64]): generated conditioned random field of solid density
+            - generated_field (list): list of randopm field values corresponding to parameter `return_property`
+            - _random_fields_vs (class:`RandomFields`): Random field generator for shear wave velocity `vs`
+            - _random_fields_rho (class:`RandomFields`): Random field generator for shear wave velocity `rho`
```

Comment:
the attributes in the docstring are not the same order as they occur in the init